.env.test

```diff
@@ -10,9 +10,11 @@ PG_FIGMA_FOR_JIRA_DB_PASSWORD=password
 DATABASE_URL='postgresql://${PG_FIGMA_FOR_JIRA_DB_ROLE}:${PG_FIGMA_FOR_JIRA_DB_PASSWORD}@${PG_FIGMA_FOR_JIRA_DB_HOST}:${PG_FIGMA_FOR_JIRA_DB_PORT}/${PG_FIGMA_FOR_JIRA_DB_SCHEMA}'
 
 FIGMA_WEB_BASE_URL=https://www.figma.com
-FIGMA_OAUTH_API_BASE_URL=https://www.figma.com
 FIGMA_API_BASE_URL=https://api.figma.com
-FIGMA_OAUTH_CLIENT_ID=test-client-id
-FIGMA_OAUTH_CLIENT_SECRET=test-client-secret
+
+FIGMA_OAUTH2_AUTHORIZATION_SERVER_BASE_URL=https://www.figma.com
```

Comment:
**note:** @jvrankul23 , @stannnous , @Roystbeef , the PR changes and adds some environment variables -- please, update your `.env` file (based on `.env.example`) when this PR is merged.

Please, let me know if can cause any issues or inconvenience.